x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/default_filters.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiButtonEmpty, EuiCode, EuiPopover, EuiText } from '@elastic/eui';
+
+interface Props {
+  seriesId: string;
+}
+export const DefaultFilters = ({ seriesId }: Props) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const onButtonClick = () => setIsPopoverOpen((isPopoverOpen) => !isPopoverOpen);
+  const closePopover = () => setIsPopoverOpen(false);
+
+  const button = (
+    <EuiButtonEmpty
+      size="xs"
+      iconType="arrowDown"
+      iconSide="right"
+      onClick={onButtonClick}
+      color="text"
+    >
+      Default filters
```

Comment:
We talked about adding the default filters like the other filters but disabled. (Could we add a tooltip to them as well? To describe why they are not editable)